test/test_core.py

```diff
@@ -4813,6 +4813,68 @@ def test_dylink_exceptions_try_catch_6(self):
 
     self.do_runf("main.cpp", "side: caught int 3\n")
 
+  @with_both_eh_sjlj
+  @needs_dylink
+  def test_dylink_exceptions_try_catch_64_bit(self):
+    self.set_setting('WASM_BIGINT')
+    create_file('main.cpp', r'''
+      #include <dlfcn.h>
+      #include <cstdint>
+      #include <iostream>
+      int main() {
+        void* handle = dlopen("liblib.so", RTLD_LAZY);
+        int64_t (*side)(int,int) = (int64_t (*)(int,int))dlsym(handle, "side");
+        try{
+          (side)(1,2);
+        }catch(int x){
+            std::cout << "Caught int " << x << std::endl;
+        }
```

Comment:
Can you do all of the try/catch stuff in the side module and simply call the side function here?

i.e. I don't think you need to try/catch across the module boundary the reproduce this issue right?

Reply:
yep